README.md

```diff
@@ -2,6 +2,12 @@
 
 For more details, check the documentation: https://docs.aws.amazon.com/codeguru/latest/profiler-ug/what-is-codeguru-profiler.html
 
+## How to use it
+
+This package is released to PyPI, so use it as any Python package from PyPI: https://pypi.org/project/codeguru-profiler-agent
+
+For a demo application that uses this agent, check our [demo application](https://github.com/aws-samples/aws-codeguru-profiler-demo-application).
```

Comment:
I think we should link to: https://github.com/aws-samples/aws-codeguru-profiler-python-demo-application

Reply:
Good catch, I will update.